lib/kubeclient/common.rb

```diff
@@ -190,8 +190,8 @@ def define_entity_methods
         end
 
         # get a single entity of a specific type by name
-        define_singleton_method("get_#{entity.method_names[0]}") do |name, namespace = nil|
-          get_entity(klass, entity.resource_name, name, namespace)
+        define_singleton_method("get_#{entity.method_names[0]}") do |name, ns = nil, opts = {}|
```

Comment:
why is `namespace` renamed to `ns` as part of this patch (and at all?)
through several code parts below it is called namespace, I don't see any reason to rename it in one place, and ns is not very readable imo anyway for newcomers to code.